code/game/mecha/equipment/tools/medical_tools.dm

```diff
@@ -316,40 +316,38 @@
 	var/mob/originaloccupant = chassis.occupant
 	spawn(0)
 		src = null //if src is deleted, still process the syringe
-		for(var/i=0, i<6, i++)
+		var/max_range = 6
+		for(var/i=0, i<max_range, i++)
 			if(!mechsyringe)
 				break
-			if(step_towards(mechsyringe,trg))

```

Comment:
github shows lots of changes here, but they are all whitespace. I.e.
```js
if(step_towards(mechsyringe,trg))
  // do stuff
```
changed to 
```js
if(!step_towards(mechsyringe,trg))
  break
// do stuff
```

Toggling "hide whitespace changes" should help make it more readable.